connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+Salesforce Connector
+====================
+
+This module allows to import and export various models between
+Odoo and Salesforce.
+
+It provides the foundations to create or extend various synchronizations.
```

Comment:
It provides as well the foundations to create or extend various synchronizations between both software.